syfertext/utils.py

```diff
@@ -168,6 +171,30 @@ def search_resource(
 
         return result[0]
 
+    # Now look into the local storage for states
+    data_path = os.path.join(str(Path.home()), "SyferText", "cache")
+
+    # Since : is a reserved character for naming files, replacing with - instead in file_name
+    tokens = query.split(":")
+    print("Tokens", tokens)
+    file_name = "-".join(tokens)
```

Comment:
```suggestion
    state_id = state_id.replace(":", "-")
```

Reply:
I need the first token as that is the directory name, so I need to tokenize either way